scss/components/status-label.scss

```diff
@@ -45,10 +45,8 @@ $fd-status-label-icon-size: fd-space(4) !default;
 
 
 .#{$block} {
-
-
-  --fd-status-label-color: var(--fd-color-text-2);
-  --fd-status-label-icon-background-color: var(--fd-color-text-2);
+  @include fd-var-size("--fd-status-label-color", $fd-color-text-2, --fd-color-text-2);
+  @include fd-var-size("--fd-status-label-icon-background-color", $fd-color-text-2, --fd-color-text-2);
```

Comment:
I don't think this is the proper use of the this mixin.  The first parameter should be the CSS property (_e.g._ `color`, `background-color`, `height`, etc.). Same for the additional uses below.